src/addtodo.js

```diff
@@ -0,0 +1,86 @@
+import { updateCheck, clearAll } from './interactive';
+
+const listParent = document.querySelector('.list');
+const returnIcon = document.querySelector('.add span');
+const textInput = document.querySelector('.add input');
+const clearButton = document.querySelector('.clear');
+
+let tasks = localStorage.getItem('todos')
+  ? JSON.parse(localStorage.getItem('todos'))
+  : [];
+
+const updateTasks = () => {
+  listParent.innerHTML = '';
+  tasks.forEach((task) => {
+    listParent.innerHTML += `
+   <li draggable ="true" id="${task.index}">
+     <div class="content">
+       <input class="check" type="checkbox" ${task.completed ? 'checked' : ''}/>
+       <input class="input" type="text" value='${task.description}' readonly />
+     </div>
+     <div class="actions">
+       <span class="material-icons drag">more_vert</span>
+       <span class="material-icons" id="deleteicon">delete_outline</span>
+     </div>
```

Comment:
Great job so far you project work perfectly weel :clap:  whatever you should only display one button at once, the ` delete_outline` button can only be displayed after making a click on the ` more_vert` button

Your:
![11](https://user-images.githubusercontent.com/57562869/149311600-039326bc-4797-4433-a5eb-92a6515c0d57.png)

Original: 
![222](https://user-images.githubusercontent.com/57562869/149311739-24aa58a5-d3e2-40bc-9eea-7b7847b9d1d4.png)